connectors/webhook/src/main/java/io/camunda/connector/inbound/HttpWebhookExecutable.java

```diff
@@ -162,6 +164,20 @@ private Function<WebhookResultContext, WebhookHttpResponse> mapResponseExpressio
     return responseExpression;
   }
 
+  private WebhookHttpResponse sanitizeBody(WebhookHttpResponse webhookHttpResponse) {
+    if (webhookHttpResponse.body() instanceof String bodyStr
+        && Pattern.compile("<//?[a-z][\\s\\S]*>").matcher(bodyStr).find()) {
```

Comment:
` Pattern.compile("<//?[a-z][\\s\\S]*>")` might be extracted as a variable to avoid creating the Pattern every time

Reply:
I don't remember what we decided, but this escaping be based on the pattern or the content-type?

Reply:
I would like the feedback of @sbuettner for this one

Reply:
What about putting something like this: 
```
  protected static Object escapeValue(Object value) {
    switch (value) {
      case String s:
        return HtmlUtils.htmlEscape(s);
      default:
        return value;
    }
  }
```

into the `InboundWebhookRestController` as this is the right layer. And call it from the `toResponseEntity` method?

Reply:
I modified it, you are right, this solution is simpler and provides a solution at a higher level.

Reply:
Great, happy to support